lib/players/human.rb

```diff
@@ -0,0 +1,27 @@
+class Player::Human < Player
+  def move(board)
+    # takes in a board argument and allows a human player to enter a move via the CLI 
+    puts "What's your move player #{token}?"
+    input = gets.strip
+     # return the value the user enters
+    if input == "quit" || input == "q" || input == "Quit" || input == "Exit" || input == "exit"
```

Comment:
I really like that you are checking for so many possible exit commands. One thing that might make this simpler would be to put them in an array.
```
exit_commands = %w(quit q exit)
if exit_commands.include?(input.downcase)
  puts "Ending game.\n"
  ...
```
You could also make `EXIT_COMMANDS` a constant at the top of your class.